### PR TITLE
load_state_dict for Normalizer, StaticNormalizer

### PR DESCRIPTION
**Pull request by wassname**
When loading a save agent you often want to load the normalizer so that it gets the inputs it expects. This makes that a little easier.

### CODE REVIEW

**Comment by ShangtongZhang** [comment]
Thanks for your contribution!
